public/tools/auto-lighthouse/index.js

```
@@ -6,71 +6,106 @@ const fs = require("fs");
 const Excel = require("exceljs");
 const lighthouse = require("lighthouse");
 const chromeLauncher = require("chrome-launcher");
+const flags = require("flags");
 
-const RESULTS_FOLDER = "./public/scan-results/data";
+flags.defineString('inputFile', null, 'File containing urls to scan.');
+flags.defineString('inputFormat', 'xls', 'Format of inputFile. Either xls or text.');
+flags.defineString('outputPrefix', '', 'Filename prefix to use when writing reports.');
+flags.defineString('ticket', 0, 'If specified, write reports to C4G-$ticket_CD_$i.json.');
+flags.defineBoolean('linux', false, 'Run on linux. Disable Chrome sandbox.');
+flags.defineBoolean('headless', false, 'Use headless chrome.');
+flags.defineBoolean('clobber', false, 'If True, overwrite existing json reports.');
```

I don't see a use of the clobber flag anywhere. Additionally, current behavior looks like it will overwrite, but the default value is false, which is misleading.

---

I did write that piece of code but somehow it got lost in the nooks and crannies of my git. Added it.